app/api/chat/route.ts

```
@@ -2,7 +2,7 @@ import { waitUntil } from "@vercel/functions";
 import { type Message } from "ai";
 import { eq } from "drizzle-orm";
 import { ReadPageToolConfig } from "@helperai/sdk";
-import { authenticateWidget, corsOptions, corsResponse } from "@/app/api/widget/utils";
+import { authenticateWidget, corsOptions, corsResponse, withAuth } from "@/app/api/widget/utils";
```

ESLint

[error] 'authenticateWidget' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)